video-object-detection/README.md

```diff
@@ -0,0 +1,30 @@
+# Video object detection example
+
+This example combines and integrates two simpler examples, the video decoder and the [deep learning server](https://github.com/veracruz-project/veracruz-examples/tree/main/deep-learning-server).
+The video decoder leverages [`openh264`](https://github.com/veracruz-project/openh264) to decode an H264 video into individual frames, which are converted to RGB and made palatable to an object detector built on top of the [Darknet neural network framework](https://github.com/mofanv/darknet-src). The output is a list of detected objects, associated with their detection probability, and an optional prediction image showing each detected object in a bounding box.
```

Comment:
Can't we have "uses" instead of "leverages"? :-)

Reply:
Wow, seems to be US slang. Will change it :+1: